mithril.js

```diff
@@ -28,6 +28,19 @@
 		return "v0.2.5"
 	}
 
+	var div = document.createElement("div")
+	var frag = document.createDocumentFragment()
+
+	function toDOM(HTML){
+		div.innerHTML = HTML
```

Comment:
Scratch that...I think the issue is that the mocks don't have an HTML parser, and so can't actually escape things.